crates/runtime/src/traphandlers/unix.rs

```diff
@@ -210,6 +211,17 @@ unsafe fn get_pc(cx: *mut libc::c_void, _signum: libc::c_int) -> *const u8 {
     }
 }
 
+unsafe fn get_fp(cx: *mut libc::c_void) -> usize {
```

Comment:
In the end this'll probably want to get folded into `get_pc` above to avoid duplicating the `cfg_if!`